_posts/2017-08-15-major-updates-coming-to-roads-data-model.md

```diff
@@ -0,0 +1,37 @@
+---
+status: publish
+layout: post
+author:
+  display_name: Greg Bunce
+  email: GBunce@utah.gov
+tags:
+  - roads
+  - data model update
+  - sgid
+  - next generation 911
+  - Federal Highway Administration ARNOLD
+published: true
+date: 2017-08-15 14:17:00
+title: 'Major update coming to the SGID Statewide Roads Data Model'
+categories:
+  - SGID Data
+---
+
+It has always been one of our goals to stay ahead of the coming changes at the NextGen 911 and FHA level, so after much statewide [coordination](https://gis.utah.gov/road-centerlines-schema-update-and-regional-workshop-notes/) and [feedback](https://gis.utah.gov/feedback-wanted-draft-statewide-road-centerlines-schema-v-3-0-x/), AGRC is officially adopting a [new statewide roads data model](https://docs.google.com/spreadsheets/d/1jQ_JuRIEtzxj60F0FAGmdu5JrFpfYBbSt3YzzCjxpfI/edit#gid=811360546). The two main drivers for this update are the Next Generation 911 GIS requirements and the Federal Highway Administration’s all roads network (ARNOLD) reporting requirements for state DOTs.
+
+AGRC will officially adopt this new roads data model on September 13th, 2017. The purpose of this blog post is to give our users a heads-up on the coming changes and to allow the necessary time to adjust your internal workflows, beforehand. After that date, we will continue to provide downloadable data (and an SDE layer) in the older model, until March 21st, 2018.  However, we encourage you to adopt this newer model sooner than later, as it will be the official statewide roads data model. There will be a subsequent blog post in September, before the official change.  
+
+**Download the New Data**  
+If you obtain the statewide roads layer from our website via the 'Road Centerlines' [download links](https://gis.utah.gov/data/transportation/roads-system/), you will now notice the new data model at the forefront. Similarly, if you are accessing the roads layer via our SDE connection (only available inside the State network) you will temporarily notice a statewide layer named 'Roads_NextGen'. This NextGen layer will replace the 'Roads' layer on September 13th, 2017. At that time, the current SDE layer named 'Roads' will be renamed to 'RoadsODM' (RoadsOldDataModel).  For both the downloadable data and the SDE layer, the older data model will be maintained and kept current, until March 21, 2018.
```

Comment:
i put the page b/c there's the shapefile and the filegeodatabase.  i can put two links (one for each).  i thought also, putting the page instead of the download links it will get users going to the page to get the data, rather than going to this post.  i can add the download links, though.  i'll have to update this page after it's published though, as those links will be created later today with the roads update post i'm working on now.  but i need this page live before i can post the road update post.  ha!  order of operation.